packages/theme_tailor/CHANGELOG.md

```diff
@@ -1,17 +1,20 @@
+# 1.0.4
+- Fix name collision when theme class contains `themes` field
```

Comment:
Currently there are more changes in the changelog, also you need to add a changelog entry in the tailor annotation and here that it is possible to load themes from build.yaml